tez-plugins/tez-protobuf-history-plugin/src/main/java/org/apache/tez/dag/history/logging/proto/HistoryEventProtoJsonConversion.java

```diff
@@ -556,12 +556,12 @@ private static JSONObject convertTaskFinishedEvent(HistoryEventProto event) thro
     events.put(finishEvent);
     jsonObject.put(ATSConstants.EVENTS, events);
 
-    long startTime = getLongDataValueByKey(event, ATSConstants.START_TIME);
+    long timeTaken = getLongDataValueByKey(event, ATSConstants.TIME_TAKEN);
 
     JSONObject otherInfo = new JSONObject();
-    otherInfo.put(ATSConstants.START_TIME, startTime);
+    otherInfo.put(ATSConstants.START_TIME, event.getEventTime() - timeTaken);
```

Comment:
actually, I saw this for all protobuf history files, that why I'm not suspecting corner case
I found that in case of a TASK_FINISHED finished event, there is always an event time (which is the end time obviously) and timeTaken in event_data, but there is no startTime there

this string is what the debugger writes for a HistoryLoggerProtos$HistoryEventProto instance while doing this conversion:
```
event_type: "TASK_FINISHED"
event_time: 1628149977709
app_id: "application_1628051798891_0030"
dag_id: "dag_1628051798891_0030_1"
vertex_id: "vertex_1628051798891_0030_1_00"
task_id: "task_1628051798891_0030_1_00_000001"
event_data {
  key: "timeTaken"
  value: "4193"
}
event_data {
  key: "status"
  value: "SUCCEEDED"
}
event_data {
  key: "numFailedTaskAttempts"
  value: "0"
}
event_data {
  key: "successfulAttemptId"
  value: "attempt_1628051798891_0030_1_00_000001_0"
}
event_data {
  key: "diagnostics"
  value: ""
}
event_data {
  key: "counters"
  value: "..."
}

```